keyboards/ergodox_ez/keymaps/lukaus/config.h

```diff
@@ -0,0 +1,20 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the ifndef and define here with just `#pragma once`, and remove the endif at the end?